src/main/java/net/mcreator/ui/modgui/ModElementGUI.java

```diff
@@ -134,7 +134,7 @@ public void setModElementCreatedListener(ModElementCreatedListener<GE> modElemen
 	@Override public ViewBase showView() {
 		MCREvent.event(new ModElementGUIEvent.BeforeLoading(mcreator, this.tabIn, this));
 
-		this.tabIn = new MCreatorTabs.Tab(this, modElement);
+		this.tabIn = new MCreatorTabs.Tab(this, modElement, false);
```

Comment:
This is now inconsistent with other tabs, such as ones from the image editor. Before, all tabs but tabs from code editor were upper-case:

![image](https://github.com/MCreator/MCreator/assets/16374228/bf7f1c62-28fc-41bd-9f64-9c13a8a4857b)

Reply:
Thinking again, uppercasing kinda gives a "signature" feel to MCreator tabs, so not sure if I want to change that. But then again, better if we move this conversation to another PR so others can chime in.

Reply:
I reverted the changes. I feel like we could keep uppercase for WORKSPACE, CONSOLE and the MAKER tabs, then normal casing for mod elements and images (the same way it's done for code files)
![tabs](https://github.com/MCreator/MCreator/assets/64546452/710fbc29-c7a1-4837-af9f-b11a733bf602)

Reply:
Although that would not be my personal preference, I am open to discussion about this in another PR if you are ready to make it :)